src/Lykke.Bil2.RabbitMq/Publication/ProhibitedRepliesMessagePublisher.cs

```diff
@@ -14,5 +14,10 @@ public void Publish<TMessage>(TMessage message, string correlationId = null)
         {
             throw new InvalidOperationException($"Replies are prohibited for this subscription. Specify replyExchangeName when subscribing using {nameof(RabbitMqEndpoint.Subscribe)}");
         }
+
+        public IMessagePublisher ChangeCorrelationId(string correlationId)
+        {
+            return this;
```

Comment:
this do not "Creates new message publisher with specified correlation id" as mentioned in interface

Reply:
It doesn't matter for this particular implementation since the only instance of this type is ProhibitedRepliesMessagePublisher.Instance and no other instance can be created since ctor is private